### ISSUE TITLE
WslService has improper Service Display Name and Description in Windows Services GUI and sc.exe

### ISSUE THREAD

**Issue by randmguy** [microsoft/WSL#9672]
### Version

Microsoft Windows [Version 10.0.19045.2604]

### WSL Version

- [X] WSL 2
- [X] WSL 1

### Kernel Version

5.15.90.1 but probably irrelevant

### Distro Version

(irrelevant): Ubuntu 20.04 lts / Ubuntu 22.04 lts / SUSE tumbleweed

### Other Software

any windows Services diagnostic tool

### Repro Steps

C:\WINDOWS\system32>wsl --update --pre-release
Es wird nach Updates gesucht.
Die neueste Version von Windows-Subsystem für Linux ist bereits installiert.

C:\WINDOWS\system32>cmd.exe /c ver

Microsoft Windows [Version 10.0.19045.2604]

C:\WINDOWS\system32>wsl --version
WSL-Version: 1.1.3.0
Kernelversion: 5.15.90.1
WSLg-Version: 1.0.49
MSRDC-Version: 1.2.3770
Direct3D-Version: 1.608.2-61064218
DXCore-Version: 10.0.25131.1002-220531-1700.rs-onecore-base2-hyp
Windows Version: 10.0.19045.2604

C:\WINDOWS\system32>sc.exe qc wslservice
[SC] QueryServiceConfig SUCCESS

SERVICE_NAME: wslservice
        TYPE               : 210  WIN32_PACKAGED_PROCESS
        START_TYPE         : 2   AUTO_START
        ERROR_CONTROL      : 0   IGNORE
        BINARY_PATH_NAME   : C:\Program Files\WindowsApps\MicrosoftCorporationII.WindowsSubsystemForLinux_1.1.3.0_x64__8wekyb3d8bbwe\wslservice.exe
        LOAD_ORDER_GROUP   :
        TAG                : 0
        DISPLAY_NAME       : ms-resource:AppName
        DEPENDENCIES       : staterepository
        SERVICE_START_NAME : LocalSystem

C:\WINDOWS\system32>sc.exe qdescription wslservice
[SC] QueryServiceConfig2 SUCCESS

SERVICE_NAME: wslservice
DESCRIPTION:  ms-resource:AppDescription

### Expected Behavior

"sc.exe qc wslservice" shows a meaningful DISPLAY_NAME ("Display Name", which is key in Windows Services GUI)
"sc.exe qdescription wslservice" shows a meaningful DESCRIPTION ("Description", which is also a field displayed in Windows Services GUI)

### Actual Behavior

"sc.exe qc wslservice" shows "ms-resource:AppName" as DISPLAY_NAME
"sc.exe qdescription wslservice" shows "ms-resource:AppDescription" as DESCRIPTION

See also issue https://codesti.com/issue/microsoft/WSL/9563
The author had this problem as a side issue during his investigation:
"The Error message there caught my eye. In Services, I can see that ms-resource:AppName is actually C:\Program Files\WindowsApps\MicrosoftCorporationII.WindowsSubsystemForLinux_1.0.3.0_x64__8wekyb3d8bbwe\wslservice.exe. So I guess this means that the WSL service crashes."

**Comment by benhillis** [comment]
Yes, this is unfortunately an issue on Windows 10. I'll follow up with the appmodel team to see if this is something they can service.
